twpca/model.py

```diff
@@ -415,6 +425,24 @@ def predict(self, X=None):
 
             return pred
 
+    def dump(self):
+        """Serializes model variables"""
+        return self._sess.run(self._vars)
```

Comment:
why not dump to a dictionary? easier for someone to figure out what the variables are, and we don't have to worry about preserving order